docs/kqueen.cli_azure_examples.rst

```diff
@@ -0,0 +1,116 @@
+Azure CLI operations with KQueen
+---------------------------------------
+
+In KQueen, you can perform a number of Azure operations using the command-line interface (CLI).
+For example, you can create a Azure account, create a new Kubernetes cluster (AKS), download the kubeconfig and use it to push an application into Kubernetes.
+
+
+To create / configure the Azure to work with KQueen:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Login to your Azure portal:
+
+.. code-block:: bash
+
+   $ az login
+   To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code CBMZ4QPTE to authenticate.
+   [
+     {
+       "cloudName": "AzureCloud",
+       "id": "8xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx2",
+       "isDefault": true,
+       "name": "Pay-As-You-Go",
+       "state": "Enabled",
+       "tenantId": "7xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxd",
+       "user": {
+         "name": "kxxxxxxxxxxxxxxxxxxxxxxxm",
+         "type": "user"
+       }
+     }
+   ]
+
+#. Create new Resource Group
+
+.. code-block:: bash
+
+   $ RESOURCE_GROUP_NAME="kqueen-demo-rg"
+   $ az group create --name "$RESOURCE_GROUP_NAME" --location westeurope
+   {
+     "id": "/subscriptions/8xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx2/resourceGroups/kqueen-demo",
+     "location": "westeurope",
+     "managedBy": null,
+     "name": "kqueen-demo",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
+#. Create the Service Principal which will have permissions to manage resources in the specified Subscription
```

Comment:
Create a service principal that will store permissions to manage resources in the a specified subscription: